pkgs/development/tools/revive/default.nix

```diff
@@ -0,0 +1,52 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "revive";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "mgechev";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-XBdid/Ze9g385dMup2Zo3RicL9k6NMjdBG6RMdsJn24=";
+    # populate values that require us to use git. By doing this in postFetch we
+    # can delete .git afterwards and maintain better reproducibility of the src.
+    leaveDotGit = true;
+    postFetch = ''
+      git -C "$out" rev-parse HEAD > $out/COMMIT
+      rm -rf "$out/.git"
```

Comment:
I'm not sure why the example I adopted this from used `find` + `xargs` to delete. I imagine this will also be sufficient